docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+app:
+  build: .
+  volumes:
+    - ./tock:/tock
+  links:
+    - db
+  working_dir: /tock
+  entrypoint: python /tock/docker_entrypoint.py
+  environment:
+    - DATABASE_URL=postgres://tock_user@db/tock
+  command: python manage.py runserver 0.0.0.0:8000
+db:
+  image: postgres:9.4
+  environment:
+    - POSTGRES_DB=tock
+    - POSTGRES_USER=tock_user
+web:
+  image: nginx
```

Comment:
Oh, upon closer inspection, it seems that the `403 Forbidden` on `/` was an issue that was never resolved: https://github.com/18F/tock/pull/333#issuecomment-198567190

Well, it works now, so yay. I'm surprised that *all* routes didn't raise `403 Forbidden` without nginx's additional dev proxy headers though... Could that actually be a security bug?